scm-ui/ui-styles/src/scm.scss

```diff
@@ -826,9 +825,6 @@ form .field:not(.is-grouped) {
     margin-bottom: 0;
   }
 }
-.menu div:first-child .menu-label {
-  background-color: $blue;
```

Comment:
There used to be a main page menu and an action page menu. Since the latter has been completely abolished, you don't need to distinguish the label background-color anymore. Moreover, we would trigger this within the component and not in the main styles.